.aws/task-definition.backend.json

```
@@ -15,11 +15,47 @@
         },
         {
           "name": "DATAPORTEN_REDIRECT_URI",
-          "value": "https://indokntnu.no/authCallback"
+          "value": "https://www.indokntnu.no/authCallback"
```

```suggestion
          "value": "https://indokntnu.no/authCallback"
```

As this is untested, i suggest to remove it